ui/src/testing/stubs/workspace-service-stub.ts

```diff
@@ -200,4 +201,20 @@ export class WorkspacesServiceStub {
       }, 0);
     });
   }
+
+  getNoteBookList(workspaceNamespace: string,
+      workspaceId: string, extraHttpRequestParams?: any): Observable<Array<FileDetail>> {
+    return new Observable<Array<FileDetail>>(observer => {
+      setTimeout(() => {
+        if (workspaceNamespace === WorkspaceStubVariables.DEFAULT_WORKSPACE_NS
+            && workspaceId === WorkspaceStubVariables.DEFAULT_WORKSPACE_ID) {
+          const fileDetailsList = [{'name': 'FileDetails', 'path': 'dummyPath'}];
```

Comment:
Is `path` here a realistic value? Isn't it going to be something like gs://bucket/notebook/foo? Probably less confusing to give a well-formed path here.

Reply:
Its a mock class so its more of a dummy value. Will change it to gs://bucket/notebook/mockFile to show the pattern kind of expected